packages/compiler/src/render3/view/compiler.ts

```diff
@@ -277,6 +274,26 @@ export function createComponentType(meta: R3ComponentMetadata): o.Type {
   return o.expressionType(o.importExpr(R3.ComponentDefWithMeta, typeParams));
 }
 
+/**
+ * Compiles the array literal of declarations into an expression according to the provided emit
+ * mode.
+ */
+function compileDeclarationListUsingMode(
```

Comment:
```suggestion
function compileDeclarationList(
```

I feel hte `...UsingMode` is redundant.